Sources/QuickTests/FunctionalTests/ItTests.swift

```diff
@@ -14,12 +14,19 @@ class FunctionalTests_ItSpec: QuickSpec {
             expect(exampleMetadata!.example.name).to(equal(""))
         }
 
-        it("has a description with セレクター名に使えない文字が入っている 👊💥") {
-            let name = "has a description with セレクター名に使えない文字が入っている 👊💥"
+        it("has a description with セレクター名に使えない文字が入っている ��") {
+            let name = "has a description with セレクター名に使えない文字が入っている ��"
             expect(exampleMetadata!.example.name).to(equal(name))
         }
 
 #if _runtime(_ObjC)
+        it("is a test with a unique name") {
+            let allSelectors: Set<String> = FunctionalTests_ItSpec.allSelectors();
+            
+            expect(allSelectors).to(contain("is_a_test_with_a_unique_name"));
+            expect(allSelectors).toNot(contain("is_a_test_with_a_unique_name_2"));
```

Comment:
This is very close, but I don't think this test is enough. I'd like a test like the following:

```swift
describe("when two examples have the exact name name") {
  it("exact same name") {}
  it("exact same name") {}

  it("uses a unique name for each") {
    expect(allSelectors.allObjects).to(equal("exact_same_name", "exact_same_name_2"));
  }
}
```